### ISSUE TITLE
NICKA nicknames

### ISSUE THREAD

**Issue by svmiller** [svmiller/codename#4]
Note to future Steve: look into this when you finally get settled in Sweden and your life isn't in the shambles/boxes it is now. This is definitely doable, though I won't bother with a content analysis of the patriotic rah-rah that the U.S. military sprinkles onto this stuff ad hoc.

http://www.designation-systems.net/usmilav/codenames.html
https://www.thedrive.com/the-war-zone/29353/how-the-pentagon-comes-up-with-all-those-secret-project-nicknames-and-crazy-code-words

I think you could mimic this in a way that's in orbit of what the U.S. government would do, and distinct from a generic grab bag, by combining the adjectives and nouns into a sample for the first word, and then the nouns for the second one. You'd have to subset to just the single word ones as well.

Basically, I think you can create a reproducible seed, which then becomes fodder for a `sample(1:101, 1)`. The result of that can be compared to the assignment blocks, creating potential options for the first word based off a `grepl()` filter that finds suitable words that match the block.

Something to think about anyway.

**Comment by svmiller** [comment]
Calling this good for now. Will assuredly update at some point.
